src/main/java/ske/mag/jenkins/buildscreen/BuildScreenView.java

```diff
@@ -151,17 +151,27 @@ protected void submit(StaplerRequest req) throws ServletException, IOException,
 
     @Override
     public Collection<TopLevelItem> getItems() {
-        return getWrappedView().getItems();
+        View wrappedView = getWrappedView();
+        if (wrappedView== null)
+        	return new ArrayList<TopLevelItem>();
+		return wrappedView.getItems();
     }
 
     @Override
     public boolean contains(TopLevelItem item) {
-        return getWrappedView().contains(item);
+        View wrappedView = getWrappedView();
+        if (wrappedView== null)
+        	return false;
+		return wrappedView.contains(item);
     }
 
     @Override
     public Item doCreateItem(StaplerRequest request, StaplerResponse response) throws IOException, ServletException {
-        return getWrappedView().doCreateItem(request, response);
+        View wrappedView = getWrappedView();
+        if (wrappedView== null){
+        	return null;

```

Comment:
Lurer på om vi bør kaste exception her som informerer om at man må wrappe et view for å få lov til å legge til nye jobber.